code/game/objects/structures/dresser.dm

```diff
@@ -51,4 +53,40 @@
 					H.socks = new_socks
 
 		add_fingerprint(H)
-		H.update_body()
\ No newline at end of file
+		H.update_body()
+
+/obj/structure/dresser/attackby(obj/item/weapon/W, mob/living/user, params)
+	add_fingerprint(user)
+	user.changeNext_move(CLICK_CD_MELEE)
+	if(iswrench(W))
+		if(anchored)
+			playsound(loc, W.usesound, 100, 1)
+			user.visible_message("[user] is loosening the [src]'s bolts.", \
+								 "<span class='notice'>You are loosening the [src]'s bolts...</span>")
```

Comment:
for anything with `[src]`, don't prefix it with `the`. This is why you don't use `[name]` to begin with; using `src`, means BYOND will handle it and use the or not, gracefully.

When you use `name`, you have to prefix it with the.